osu.Game.Rulesets.Osu/Skinning/FollowCircle.cs

```diff
@@ -13,90 +14,99 @@ namespace osu.Game.Rulesets.Osu.Skinning
 {
     public abstract partial class FollowCircle : CompositeDrawable
     {
-        [Resolved]
-        protected DrawableHitObject? ParentObject { get; private set; }
+        protected DrawableSlider? DrawableObject { get; private set; }
+
+        private readonly IBindable<bool> tracking = new Bindable<bool>();
 
         protected FollowCircle()
         {
             RelativeSizeAxes = Axes.Both;
         }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(DrawableHitObject? hitObject)
         {
-            ((DrawableSlider?)ParentObject)?.Tracking.BindValueChanged(tracking =>
```

Comment:
This may look bad but it's not that bad because `FollowCircle` has a 1:1 relationship with a `DrawableHitObject`. This was subscribing twice somehow, but "only" twice.